frontend/src/container/JoinWorkspace.jsx

```diff
@@ -134,6 +142,23 @@ export class JoinWorkspace extends React.Component {
     this.setState({ filter: filter.toLowerCase() })
   }
 
+  async handleClickRequestAccess (space, userId) {
+    const { props } = this
+    const fetchPutUserSpaceSubscription = await props.dispatch(putUserWorkspaceSubscription(space.id, userId))
+
+    switch (fetchPutUserSpaceSubscription.status) {
+      case 200: props.dispatch(newFlashMessage(props.t(
+        'Your request to join {{space}} will be handled by a space manager. The result will be shown on the notification wall.',
+        {
+          space: space.label,
+          interpolation: { escapeValue: false }
```

Comment:
Is this necessary?

Reply:
the space label can contain anything, including html tags... whether or not this is necessary depends on if we want to disable i18next interpolation (escaping HTML code). do you prefer enable?

Reply:
I'd avoid disabling escaping unless deemed strictly necessary, since this can cause security issues.
Here, formatting in space name do not seem relevant so I'd display space as text, and if it happens to contain html tags, show them plain text. What do you think?

Reply:
I don't have preferences, I just like it to be coherent